lms/djangoapps/instructor/permissions.py

```diff
@@ -15,6 +15,11 @@
 ENABLE_CERTIFICATE_GENERATION = 'instructor.enable_certificate_generation'
 GENERATE_CERTIFICATE_EXCEPTIONS = 'instructor.generate_certificate_exceptions'
 GENERATE_BULK_CERTIFICATE_EXCEPTIONS = 'instructor.generate_bulk_certificate_exceptions'
+GENERATE_EXAMPLE_CERTIFICATES = 'instructor.generate_example_certificates'
```

Comment:
Is `GENERATE_EXAMPLE_CERTIFICATES` still viable these days? I thought generating example certificates were related to PDF certificates, and this functionality has been gutted from edx-platform for quite some time.